packages/oui-file/src/file.html

```diff
@@ -0,0 +1,132 @@
+<!-- Controls -->
+<input type="hidden"
+    ng-attr-id="{{::$ctrl.id}}"
+    ng-attr-name="{{::$ctrl.name}}"
+    ng-disabled="$ctrl.disabled"
+    ng-required="$ctrl.required"
+    ng-model="$ctrl.model">
+<input class="oui-file__input" type="file"
+    ng-if="::!$ctrl.attachments"
+    ng-attr-id="{{::$ctrl.selectorId}}"
+    ng-attr-accept="{{::$ctrl.accept}}"
+    ng-attr-size="{{::$ctrl.size}}"
+    ng-disabled="$ctrl.disabled"
+    tabindex="-1">
+<input class="oui-file__input" type="file"
+    ng-if="::$ctrl.attachments"
+    ng-attr-id="{{::$ctrl.selectorId}}"
+    ng-attr-accept="{{::$ctrl.accept}}"
+    ng-attr-size="{{::$ctrl.size}}"
+    ng-disabled="$ctrl.disabled"
+    tabindex="-1"
+    multiple>
+<!-- /Controls -->
+
+<!-- Selector -->
+<div class="oui-file-selector"
+    ng-if="::!$ctrl.multiple && !$ctrl.droparea"
+    ng-class="{
+        'oui-file-selector_error': $ctrl.form && $ctrl.form[$ctrl.name].$invalid && ($ctrl.form[$ctrl.name].$touched || $ctrl.form.$submitted)
+    }">
+
+    <!-- Fake Selector -->
+    <div class="oui-file-selector__input">
+        <button class="oui-file-selector__label" type="button"
+            ng-blur="$ctrl.setInputTouched()"
+            ng-click="$ctrl.openFileSelector()"
+            ng-disabled="$ctrl.disabled">
+            <span class="oui-file-selector__placeholder"
+                ng-if="!$ctrl.model[0]"
+                ng-bind="::$ctrl.placeholder">
+            </span>
+            <span class="oui-file-ellipsis"
+                ng-if="!!$ctrl.model[0]">
+                <span class="oui-file-ellipsis__name" ng-bind="$ctrl.constructor.getFileName($ctrl.model[0])"></span>
+                <span class="oui-file-ellipsis__extension" ng-bind="$ctrl.constructor.getFileExtension($ctrl.model[0])"></span>
+                <span class="oui-file-ellipsis__size" ng-bind="$ctrl.getFileSize($ctrl.model[0])"></span>
+            </span>
+        </button>
+        <button class="oui-file__remove" type="button"
+            ng-show="!!$ctrl.model[0]"
+            ng-click="$ctrl.resetFile()"
+            ng-disabled="$ctrl.disabled">
+            <span class="oui-icon oui-icon-close_thin"></span>
```

Comment:
`aria-hidden`

Reply:
Alternative text for screen readers

Reply:
Why hidden, since it's the close button icon and its only content ?
I'll check for the `aria-label`.

Reply:
I've got no away to know if it's a close button or a delete my account button or a like on facebook button if I'm on a screen reader

Reply:
done